nixos/modules/tasks/network-interfaces.nix

```diff
@@ -565,6 +582,7 @@ in
             example = 100;
             type = types.nullOr types.int;
             description = ''
+              DEPRECATED, use `driverOptions`.
```

Comment:
So we have a LACP bond that is configured with the deprecated options but after this change came up with balance-rr instead. Which broke its networking. :disappointed:

This should either trigger an error or merging should work. As this breaks peoples' setups I'm inclined to revert this for now.

:+1: on the change, though.